src/epoch/initializers.rs

```diff
@@ -341,6 +341,20 @@ impl Epoch {
         Self::from_duration(Duration::from_parts(0, nanoseconds), TimeScale::BDT)
     }
 
+    #[must_use]
+    /// Initialize an Epoch from the provided IEEE 1588-2008 (PTPv2) second timestamp since TAI midnight 1970 January 01.
+    /// PTP uses the TAI timescale but with the Unix Epoch for compatibility with unix systems.
+    pub fn from_ptp_seconds(seconds: f64) -> Self {
+        Self::from_tai_duration(UNIX_REF_EPOCH.to_tai_duration() + seconds * Unit::Second)
+    }
+
+    #[must_use]
+    /// Initialize an Epoch from the provided IEEE 1588-2008 (PTPv2) millisecond timestamp since TAI midnight 1970 January 01.
+    /// PTP uses the TAI timescale but with the Unix Epoch for compatibility with unix systems.
+    pub fn from_ptp_milliseconds(millisecond: f64) -> Self {
```

Comment:
Minor thought,

(I also like having construction methods like these a lot, they make life much easier for the end user)
but since PTP applications are usually located around tens of nanoseconds (usually) I would rather have `from_ptp_nanoseconds` instead of milliseconds ?